### PR TITLE
fix: Home page on the teacher's view 

### LINKED ISSUE #119: Home page on the teacher's view
Courses - ClassroomIO - Google Chrome 12_19_2023 10_33_31 PM (2)

### Improvement Description

There are two expandable icons next to the name of the organization when it should be just one, the dropdown icon.

### Rationale

Typically the two expandable icon gives off a reduction or addition action at first glance and that is not what it's intended to.

### Proposed Solution

Checkout the icon from [carbon-icon-svelte](https://carbon-icons-svelte.onrender.com/) and replace the current icon with this one.

image

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this improvement suggestion.

- [X] I have searched the existing issues and improvement suggestions to avoid duplication.
- [X] I have provided a clear description of the improvement being suggested.
- [X] I have explained the rationale behind this improvement.
- [x] I have included any relevant technical details or design suggestions.
- [X] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by anishyadavv**
## What does this PR do?
Fixed the caret icon on sidebar Component which is used to change org.
Fixes #119 

caret

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

Change can be seen on sidebar on dashboard route

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [x] Ran `pnpm build`
- [x] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [x] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

File: apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte [hunk 1]
```diff
@@ -37,7 +37,7 @@
       <p class="dark:text-white text-lg font-bold whitespace-nowrap truncate">
         {$currentOrg.name}
       </p>
-      <CaretSortIcon size={16} class="ml-2" />
+      <CaretDown size={16} class="ml-2"/>
```
Comment (rotimi-best) [inline apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte]:
```suggestion
      <CaretDown size={16} class="ml-2" />
```

**Review by rotimi-best — COMMENTED** [review]
Thank you for this. Really appreciate it 😊. Just one more change needed. 🚀

File: apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte (hunk 1 above)
Comment (anishyadavv) [reply apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte]: Done with change @rotimi-best

Comment (rotimi-best) [reply apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte]: Thanks a lot for this.

Can you please share your Twitter handle so I can give you a shout out

**Review by rotimi-best — APPROVED** [review]
Looks really good. Thank you very much 😊

File: apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte (hunk 1 above)
Comment (anishyadavv) [reply apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte]: sure @rotimi-best , My twitter -  https://twitter.com/AnishYadav32381

**Pull request merged**